openlibrary/core/lending.py

```diff
@@ -36,9 +36,9 @@
 #     BookReader loan status is always current.
 LOAN_FULFILLMENT_TIMEOUT_SECONDS = 60*5
 
-IA_API_URL = None
-AVAILABILITY_API = None
-IA_XAUTH_API_URL = None
+config_ia_api_url = None
```

Comment:
Totally nitpicky, and sorry for not commenting earlier, but `config_ia_api_url` is a little too generic a name: the "Internet Archive API URL" begs "Umm... API for what?" The others are clear.

Everything else looks great!